src/snappy_device_agents/devices/multi/multi.py

```diff
@@ -146,14 +146,14 @@ def create_jobs(self):
             try:
                 job_id = self.client.submit_job(job)
             except OSError as exc:
-                logger.exception("Unable to create job: %s", job_id)
+                logger.exception("Unable to create job: %s", job)
```

Comment:
You're right, we can't return the job_id here, but we shouldn't return the entire job json either. Perhaps the queue name that it would have gone on, and say "Unable to create job in queue: {...}"?

Reply:
That makes sense, I did not realize that we'd get the entire job json here.
Your other suggestion sounds reasonable to me - I'll try and implement this shortly.

Reply:
Were you still working on this patch?